ui/components/app/assets/asset-list/cells/asset-cell-badge.tsx

```diff
@@ -38,7 +38,7 @@ export const AssetCellBadge = React.memo(
             size={AvatarNetworkSize.Xs}
             name={allNetworks?.[chainId as Hex]?.name}
             src={badgeWrapperSrc}
-            backgroundColor={BackgroundColor.backgroundMuted}
+            backgroundColor={BackgroundColor.backgroundSection}
```

Comment:
Replacing any `background/muted` with `background/section` if it's not interactive. The color should be approximately equivalent to what `background/muted` was previously.